proto/p4/v1/p4runtime.proto

```diff
@@ -210,6 +210,22 @@ message TableEntry {
   IdleTimeout time_since_last_hit = 10;
   // Arbitrary metadata from the controller that is opaque to the target.
   bytes metadata = 11;
+  // const is true only for entries that cannot be deleted or
+  // modified, e.g. those in a table with `has_initial_entries = true`
+  // that have been marked `const` _on that individual entry_ in the
+  // P4 source code within its `entries` table property.
+  //
+  // TODO: Should const be true for all entries read from a table
+  // declared with `const entries`?  Would that be considered a
+  // backwards-compatible change in this specification?  If it is
+  // considered backwards-incompatible, is it an acceptable
+  // incompatibility?
+  //
+  // TODO: Should const be true when reading a default entry from a
+  // table, if it was declared as `const default_action` in the P4
+  // source code?  Would that be considered a backwards-compatible
+  // change in this specification?
+  bool const = 13;
```

Comment:
Is this field actually writable? It seems like it's an invariant based on the P4 code itself. Do we want to be able to write it back (with correct value) just for the sake of read-write symmetry? I'm trying to comprehend the new para. around line 3046 in the .mdk spec itself:

>The `const` field must be `false` in any `INSERT`, `MODIFY`, or
`DELETE` write request of a table entry.  If it is true, the server
must reject the operation and return an `INVALID_ARGUMENT` error.